projects/display-projects.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>View Project</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="../helpers/css/components.css" />
+    <link rel="stylesheet" href="project.css" />
+  </head>
+  <body>
+
+    <div class="container-fixed" >
+        <div class="project-display" id="view-project-container">
+            <!--Render using javascript -->
```

Comment:
Create the elements for each project detail in the html and target them in javascript by ID to render their contents.